pages/clusters/manage_clusters.md

```diff
@@ -93,14 +93,16 @@ If you migrate all your existing agents over to clusters, make sure to add all y
 
 ## Pause a queue
 
+> 📘 Enterprise feature
+> Cluster queue pausing is only available to [Pro and Enterprise](https://buildkite.com/pricing) plans.
```

Comment:
```suggestion
> Cluster queue pausing is only available on [Pro and Enterprise](https://buildkite.com/pricing) plans.
```

If it's "to" then I think it needs to be about the user: "to <> plan customers."